scenarioo-server/build.gradle

```diff
@@ -24,6 +24,47 @@ apply plugin: 'eclipse-wtp'
 apply from: "${rootProject.projectDir}/build-java.gradle"
 apply plugin: 'org.springframework.boot'
 
+apply plugin: 'cz.habarta.typescript-generator'
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath group: 'cz.habarta.typescript-generator', name: 'typescript-generator-gradle-plugin', version: '2.13.489'
+    }
+}
+generateTypeScript {
+    classes = [
```

Comment:
Cant you move all the backend DTOs into one package (`org.scenarioo.model`) and then import it from there? 

Listing every single file is dangerous imo as you expect new code to appear in your frontend Dtos as well.

Reply:
Unfortunately I would change the backend a lot. I don't want to do that. I'm aware that this is far from perfect but if somebody moves code, the build will break. So we should notice it.